exorelay-javascript/package.json

```diff
@@ -4,27 +4,27 @@
   "author": "Kevin Goslar",
   "dependencies": {
     "chalk": "1.1.3",
-    "debug": "2.6.0",
+    "debug": "2.6.6",
     "rails-delegate": "0.6.2",
     "uuid": "3.0.1",
     "wait": "0.1.0",
-    "ws": "2.0.3"
+    "ws": "2.3.1"
   },
   "description": "Communication relay between JavaScript code bases and the Exosphere environment",
   "devDependencies": {
     "chai": "3.5.0",
     "cucumber": "1.3.1",
     "cucumber-snippets-livescript": "1.0.1",
     "dependency-lint": "4.3.1",
-    "ejs": "2.5.5",
+    "ejs": "2.5.6",
     "exocom-mock": "0.21.4",
```

Comment:
Good point. I have created https://github.com/Originate/exosphere-sdk/issues/255 to track this!